.github/workflows/windows-msys2.yml

```diff
@@ -86,21 +86,19 @@ jobs:
 #          set -x
 #          ci/install_cacheable_dependencies.sh
       - name: download googletest
+        uses: actions/checkout@v3
+        with:
+          repository: google/googletest
+          path: googletest-src
+          fetch-depth: 0
```

Comment:
`fetch-depth: 0` means fetch all, maybe you meant `fetch-depth: 1`?

Reply:
Using fetch-depth:1 means that we use the latest commit on main branch.
googletest latest commit on main brunch is usually broken.

The code that you had here 
```
          git clone --no-checkout "https://github.com/google/googletest.git" "googletest-src"
          cd googletest-src
          git checkout c43f710 -
```
also fetched full history and checked out a stable verion, not the last

Reply:
Ahh got it. I thought we no longer had to fix to a version... it's just strange to fix to a commit instead of to a tag. Thanks!